runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1246,6 +1266,41 @@ class VM_JFRConstantPoolTypes {
 		gcConfiguration->heapAddressBits = J9JAVAVM_REFERENCE_SIZE(vm) * 8;
 	}
 
+	uintptr_t recordSystemProcessEvent(uintptr_t pid, const char *commandLine, void *userData)
+	{
+		VM_JFRConstantPoolTypes *constantPool = static_cast<VM_JFRConstantPoolTypes *>(userData);
+		J9Pool *systemProcessTable = constantPool->getSystemProcessTable();
+
+		SystemProcessEntry *entry = static_cast<SystemProcessEntry *>(pool_newElement(systemProcessTable));
+		if (NULL == entry) {
+			return (uintptr_t)-1;
+		}
+		entry->ticks = j9time_nano_time();
+		char pidString[32];
+		snprintf(pidString, sizeof(pidString), "%lu", (unsigned long)pid);
+		entry->pid = strdup(pidString); 
+		entry->commandLine = strdup(commandLine);
```

Comment:
also add a counter and keep track of the length of the string (`+= strlen(commandLine)`) so `requiredBufferSize` can be more accurate